app/cdap/components/ParsingDrawer/index.tsx

```diff
@@ -139,7 +141,11 @@ export default function({ setLoading, updateDataTranformation }) {
           handleCloseError={() =>
             setErrorOnTransformation({
               open: false,
-              message: 'Error Encountered...',
+              message: JSON.stringify(
```

Comment:
can we use toString() method rather than JSON.stringify

Reply:
resolved